src/components/FlowRunCreateForm.vue

```diff
@@ -109,6 +109,7 @@
     deployment: Deployment,
     // these must be the unmapped SchemaValues
     parameters?: SchemaValues,
+    initName?: string,
```

Comment:
Can we just call this prop "name"? We haven't used an "init" prefix anywhere else and the parameters prop does the same thing and doesn't have a prefix.

Reply:
Ah I guess I thought it would conflict with the const called `name` but that doesn't make sense - good call, will update

Reply:
updated 👍

Reply:
Thanks!